src/Mod/Path/PathScripts/PathSurface.py

```diff
@@ -249,8 +249,9 @@ def opPropertyDefaults(self, obj, job):
             'AvoidLastX_Faces': 0,
             'PatternCenterCustom': FreeCAD.Vector(0.0, 0.0, 0.0),
             'GapThreshold': 0.005,
-            'AngularDeflection': 0.25,
-            'LinearDeflection': 0.0001,
+            'AngularDeflection': 0.25, # AngularDeflection is unused
```

Comment:
It has been unused for a long time (years?). I mentioned that we should remove it in the tessellation PR, but I think we should do so consistently for both waterline and 3d surface.

Reply:
When I suggest remove `AngularDeflection`, I am referring to it as a property of the operation, and only within the scope of the PathSurface and PathSurfaceSupport modules.  If we have no calculations dependent upon the `obj.AngularDeflection` property within the targeted modules, we can delete it within the same.
I am the one who added it a while back in order to expose it to the users, so they could adjust the mesh generation settings.  If we are not using it now, let's get rid of it, unless we see it might come in to play in the near future for additional improvements to the operation. 
((For example: There is a lot of geometry selection and manipulation code in the Waterline module, cloned from 3D Surface. I left it there because I intend to apply and utilize it for Waterline, to allow face selected waterline operations.))

Reply:
Waterline OCL should have the same tessellation requirements as linear OCL. I haven't checked whether it is indeed using the same tessellation code right now, but if not then we should probably make it so, and unify the preferences as well.

In any case, I think it makes sense to do so in a separate PR.

Reply:
> Waterline OCL should have the same tessellation requirements as linear OCL. I haven't checked whether it is indeed using the same tessellation code right now, but if not then we should probably make it so, and unify the preferences as well.
Agreed.

> In any case, I think it makes sense to tackle the AngularDeflection / tessellation cleanup in a separate PR.
Agreed.

wfm ;)